Examples/RecordRowExample.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using InfluxDB.Client;
+using InfluxDB.Client.Api.Domain;
+
+
+namespace Examples
+{
+    public static class RecordRowExample
+    {
+        public static async Task Main()
+        {
+            const string url = "http://localhost:9999/";
+            const string username = "my-user";
+            const string password = "my-password";
+            const string bucket = "my-bucket";
+            const string org = "my-org";
+
+            using var client = InfluxDBClientFactory.Create(url, username, password.ToCharArray());
```

Comment:
Please use authorisation via token, its preferred way:
```suggestion
            using var client = InfluxDBClientFactory.Create(url, token.ToCharArray());
```